mvvm/src/iosMain/kotlin/dev/icerock/moko/mvvm/binding/LiveDataBindingExt.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+package dev.icerock.moko.mvvm.binding
+
+import dev.icerock.moko.mvvm.livedata.LiveData
+import dev.icerock.moko.mvvm.livedata.map
+import dev.icerock.moko.resources.StringResource
+import dev.icerock.moko.resources.desc.StringDesc
+import dev.icerock.moko.resources.desc.desc
+
+fun <T> LiveData<T>.descOrEmpty(): LiveData<StringDesc> {
+    return this.map {
+        when (it) {
+            is StringDesc -> it
+            is StringResource -> it.desc()
+            is String -> it.desc()
+            else -> "".desc()
+        }
+    }
+}
```

Comment:
as i see it just one case of many others with generics issue of optional types. i think we should correct give optionals in api of ios